docs/scripts_md/MRI.md

```diff
@@ -0,0 +1,366 @@
+# NAME
+
+NeuroDB::MRI -- A set of utility functions for performing common tasks
+relating to MRI data (particularly with regards to registering MRI
+files into the LORIS system)
+
+# SYNOPSIS
+
+    use NeuroDB::File;
+    use NeuroDB::MRI;
+    use NeuroDB::DBI;
+
+    my $dbh = NeuroDB::DBI::connect_to_db();
+
+    my $file = NeuroDB::File->new(\$dbh);
+
+    $file->loadFileFromDisk('/path/to/some/file');
+    $file->setFileData('CoordinateSpace', 'nonlinear');
+    $file->setParameter('patient_name', 'Larry Wall');
+
+    my $parameterTypeID = $file->getParameterTypeID('patient_name');
+    my $parameterTypeCategoryID = $file->getParameterTypeCategoryID('MRI Header');
+
+# DESCRIPTION
+
+Really a mishmash of utility functions, primarily used by process\_uploads and
```

Comment:
is `process_uploads` some old file? I think we should update this to be `tarchiveLoader` perhaps?

Reply:
I have absolutely no idea... I guess we could replace this by tarchiveLoader?

Reply:
I think so.
check the undoUpload scipt; it seems to be dealing mainly with anything after the dicomTar.pl step (i.e. not touching the tarchive_* tables and deleting anything after that) which to me implies that what was referred to in the past as Upload is actually our tarchiveLoader...
maybe we should bring this up at the Imaging meeting today.